modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,398 @@
+
+//TODO: clarify name (SATHISH)
+Young Gen GC Duration::
+Amount of time taken for young generation Java garbage collection
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Committed::
+Amount of physical memory committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Heap Usage::
+Amount of physical memory used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Thread Count::
+//TODO: threads that are available or used??? (SATHISH)
+Number of threads for simultaneous requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Committed Virtual Memory::
+Amount of memory (in MB) committed to the JVM
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+CPU Utilization::
+CPU work as a percentage of total CPU time
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::app-detail-jvm[]
+//------------------------------------------------------//
+
+//========================//
+//API DETAILS: ALL METRICS//
+//========================//
+//
+// API DETAILS: Header-level metrics
+//-------------------------
+//tag::api-detail-header[]
+
+//TODO: change to API (SATHISH)
+Type::
+API implementation type (*Api*)
+
+Status::
+Status code that indicates whether the governance policies and SLA tiers are functional (*Active*) or not
++
+For all codes, see information on xref:api-manager::latest-overview-concept.adoc#what-api-manager-looks-like[tracking registration status] in the API Manager UI documentation.
+
+Implementation URL:: 
+URL of deployed API integration
+
+//TODO: suggest Runtime Version conform on Runtime (App) vs. Runtime Version (API)
+//TODO:what If the runtime version is empty... (SEE BOOKS API IN STGX)
+//TODO: capitalize "version"?
+//(SATHISH)
+Runtime version::
+Mule Gateway or Flex Gateway runtime version used by the API implementation 
+
+//  shared header metric
+include::partial$include-insights-charts.adoc[tag=entity-replicas]
+
+//TODO_TODO_TODO: What if empty but seems to be running?
+API Instance ID::
+Instance identifier for an API implementation incorporated into a Flex Gateway or API Gateway runtime
+
+//TODO: capitalize "version"?
+API version::
+Version of the API implementation
++
+For more information, see xref:api-manager::api-instance-landing-page.adoc[].
+//end::api-detail-header[]
+//-----------------------//
+//
+//-------------------------
+// API DETAILS: Overview tab
+//-------------------------
+//tag::api-detail-overview[]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="average-latency"]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="p95-latency"]
+
+//shared Overview metric
+include::partial$include-insights-charts.adoc[tag="total-requests"]
+
+//TODO_TODO: to avoid duplication (see chart with same name this one) CHANGE NAME to something like Policy Violation Volume (or similar)
```

Comment:
```suggestion
//TODO_TODO: to avoid duplication (see chart with same name this one) CHANGE NAME to something like Policy Violation Volume (or similar)
```
@sathishpaulleo